src/commands/math/commands.ts

```diff
@@ -1704,12 +1706,6 @@ class Bracket extends DelimsNode {
     this.getEnd(L).deleteOutOf = function (dir: Direction, cursor: Cursor) {
       (this.parent as Bracket).deleteSide(dir, true, cursor);
     };
-    // FIXME HACK: after initial creation/insertion, finalizeTree would only be
-    // called if the paren is selected and replaced, e.g. by LiveFraction
-    this.finalizeTree = this.intentionalBlur = function () {
-      this.delimFrags[this.side === L ? R : L].removeClass('mq-ghost');
-      this.side = 0;
-    };
```

Comment:
This is too harsh. Now, the student _must_ write the closing bracket pair for the ghost to be removed. If I remember correctly, we want to disable the behavior where lose-of-focus would cause the bracket pair to autocomplete such that the student can use the formula-editor to compose bracket pairs.

I believe this would already be achieved if we only remove the `this.intentionalBlur` binding here, but keep the `this.finalizeTree` assignment intact.

```
    // FIXME HACK: after initial creation/insertion, finalizeTree would only be
    // called if the paren is selected and replaced, e.g. by LiveFraction
    this.finalizeTree = function () {
      this.delimFrags[this.side === L ? R : L].removeClass('mq-ghost');
      this.side = 0;
    };
```